devices/xiaomi.js

```diff
@@ -1202,8 +1202,7 @@ module.exports = [
                 .withFeature(exposes.enum('state', ea.STATE_SET, ['control_relay', 'decoupled']))
                 .withEndpoint('right'),
             e.action(['single_left', 'double_left', 'single_right', 'double_right', 'single_both', 'double_both']),
-            e.power_outage_memory().withEndpoint('left'),
-            e.power_outage_memory().withEndpoint('right'),
+            e.power_outage_memory(),
```

Comment:
Involving @ruifung  here.

I don't understand this, does this mean the power outage memory of the buttons cannot be configured individually?

Reply:
yes, the setting is general for the entire device and does not apply to a specific button
![image](https://user-images.githubusercontent.com/8360230/124971335-b9ca6300-e031-11eb-853a-17fc916f7ede.png)

Reply:
changing this attribute affects both switch channels.

Reply:
> 
> 
> Involving @ruifung here.
> 
> I don't understand this, does this mean the power outage memory of the buttons cannot be configured individually?

I don't have this model of switch, so I can't say much but...

To my view the better question is, which endpoints is this cluster actually on though.

If it currently works for the entire switch regardless of if it's sending to endpoint 1 or 2, then exposing it without a specific endpoint would be an accurate representation.

If it *can* be controlled individually, and the xiaomi app/gateway just does magic behind the scenes then it's fine to leave it as is.